cemu/ui/utils.py

```diff
@@ -65,3 +66,45 @@ def popup(msg: str, type: PopupType = PopupType.Error, title: str = ""):
 
     dbg(f"{title} - {msg}")
     QMessageBox(icon, title, msg, buttons=QMessageBox.StandardButton.Discard).exec()
+
+
+def is_dark_mode(palette: QPalette):
```

Comment:
missign types a bit everywhere 

```suggestion
def is_dark_mode(palette: QPalette) -> int:
```